misc/python/materialize/test_analytics/data/feature_benchmark/feature_benchmark_result_storage.py

```diff
@@ -73,7 +72,7 @@ def add_result(
                     {as_sanitized_literal(result_entry.scale)},
                     {result_entry.is_regression},
                     {result_entry.wallclock.result or 'NULL::DOUBLE'},
-                    {result_entry.messages.result or 'NULL::INT'},
+                    None,  # to be removed when we remove the "messages" column
```

Comment:
I don't think this will work.

```suggestion
                    'NULL::INT',  # to be removed when we remove the "messages" column
```

Reply:
Oops, of course.